src/All.php

```diff
@@ -0,0 +1,155 @@
+<?php namespace Leean\Acf;
+
+/**
+ * Class to provide helpers for getting a all ACF fields for an entity.
+ */
+class All
+{
+	/**
+	 * Is the ACF plugin active.
+	 *
+	 * @return bool
+	 */
+	public static function is_active() {
+		return function_exists( 'get_field_object' );
+	}
+
+	/**
+	 * Apply the default transforms if any exist for this field type.
+	 *
+	 * @param array $field The field.
+	 * @return mixed
+	 */
+	private static function apply_default_transform( $field ) {
+		$class = '\\' . __NAMESPACE__ . '\\Transforms\\' . str_replace( '_', '', ucwords( $field['type'], '_' ) );
```

Comment:
I think so but I think a method should be even better since at the moment is a little bit hard to follow the code in that single line. 

> El 21/03/2016, a las 12:47 p.m., Adam Fenton <notifications@github.com> escribió:
> 
> In src/All.php:
> 
> > +	 * Is the ACF plugin active.
> > +	 *
> > +	 * @return bool
> > +	 */
> > +	public static function is_active() {
> > +		return function_exists( 'get_field_object' );
> > +	}
> > +
> > +	/**
> > +	 * Apply the default transforms if any exist for this field type.
> > +	 *
> > +	 * @param array $field The field.
> > +	 * @return mixed
> > +	 */
> > +	private static function apply_default_transform( $field ) {
> > +		$class = '\\' . __NAMESPACE__ . '\\Transforms\\' . str_replace( '_', '', ucwords( $field['type'], '_' ) );
> Would $class_name be better?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>